### PR TITLE
Use the MCS SIG APIs instead of our own

### PR DESCRIPTION
**Pull request by aswinsuryan**
Fixes:submariner-io/lighthouse#336

Signed-off-by: Aswin Surayanarayanan <asuryana@redhat.com>

### CODE REVIEW

**Review by skitt — CHANGES_REQUESTED** [review]
The Dockerfile shouldn’t be part of this PR; apart from that, LGTM!

You’ll also need to add the MCS YAML files to the linting exclusion list in `.yamllint.yml`, and it might be worth separating the old MCS removal in a separate PR.

**Comment by mangelajo** [comment]
@aswinsuryan @vthapar question independent to this specific patch. At this point, where we are starting to care about updates, probably in Lighthouse we may want to look for old "ServiceExports", converting them to new services exports, and deleting the old ones. So existing deployments are not affected.

File: Dockerfile.dapper783451420 [hunk 1]
```diff
@@ -0,0 +1,22 @@
+FROM quay.io/submariner/shipyard-dapper-base:devel
```
Comment (mangelajo) [inline Dockerfile.dapper783451420:1]: This file should not be added :)

**Review by mangelajo — CHANGES_REQUESTED** [review]
Looks good, just removing the dockerfile from dapper.

**Comment by mangelajo** [comment]
Ahh I see @skitt already commented about the dockerfile ':D

File: Dockerfile.dapper783451420 (hunk 1 above)
Comment (aswinsuryan) [reply Dockerfile.dapper783451420]: Done . it was due to a  `git add .` . :)

**Comment by aswinsuryan** [comment]
> The Dockerfile shouldn’t be part of this PR; apart from that, LGTM!
> 
> You’ll also need to add the MCS YAML files to the linting exclusion list in `.yamllint.yml`, and it might be worth separating the old MCS removal in a separate PR.

WIll make MCS removal a separate PR.

**Review by skitt — APPROVED** [review]

**Review by vthapar — APPROVED** [review]

**Review by tpantelis — APPROVED** [review]

**Review by mangelajo — APPROVED** [review]

**Pull request merged**